### DISCUSSION TITLE
[Proposal] WasmEdge Community Meeting

### DISCUSSION THREAD

**Discussion by alabulei1** [WasmEdge/WasmEdge#903] [General]
Hello Community,

WasmEdge maintainers are going to start the WasmEdge Community meeting via video meeting in October 2021. It will be a monthly recurring meeting on the first Tuesday at 8 AM of each month (PST).

We will discuss the status of the WasmEdge project and WebAssembly, and listen to the feedback or suggestion from the WasmEdge developer community carefully. Anyone interested in WebAssembly and WasmEdge is welcome to join the monthly meeting. 

We will use [a public google doc](https://docs.google.com/document/d/1iFlVl7R97Lze4RDykzElJGDjjWYDlkI8Rhf8g4dQ5Rk/edit?usp=sharing) to manage the meeting information, like the specific time, agenda, joining, and meeting notes.

> Join [the CNCF WasmEdge community](https://community.cncf.io/wasmedgeruntime-community/) to get the notification.

Feel free to comment your thoughts on this issue. Any suggestion is welcome. Looking forward to hearing from you!

**Comment by shivay-fabric** [comment]
This is a really great addition

**Comment by alabulei1** [comment]
Hi Community,

Our first meeting will begin at 8 AM, 5th October(PST). You can find more details via google doc. Feel free to add your item.

https://docs.google.com/document/d/1iFlVl7R97Lze4RDykzElJGDjjWYDlkI8Rhf8g4dQ5Rk/edit?usp=sharing

Looking forward to hearing from you!

**Comment by shivay-fabric** [comment]
@alabulei1 I wanted to add a topic. can I get access to the doc

**Comment by alabulei1** [comment]
> @alabulei1 I wanted to add a topic. can I get access to the doc

No problem. Please join the WasmEdge Email group, and you will get access.  https://groups.google.com/g/wasmedge/
